ci-operator/step-registry/aro/ocm/register/aro-ocm-register-commands.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+export KUBECONFIG=${SHARED_DIR}/kubeconfig
+PULL_SECRET_FILE=${PULL_SECRET_FILE:="${CLUSTER_PROFILE_DIR}/pull-secret"}
+CLOUD_PULL_SECRET=$(jq '{ "cloud.openshift.com": .auths["cloud.openshift.com"] }' ${PULL_SECRET_FILE})
+UPDATED_PULL_SECRET_FILE="${SHARED_DIR}/pull-secret-new"
+CLUSTER_PULL_SECRET="${SHARED_DIR}/cluster-pull-secret"
+CLUSTER_UUID=$(oc get clusterversion version -o jsonpath='{.spec.clusterID}{"\n"}')
+oc get secrets pull-secret -n openshift-config -o template='{{index .data ".dockerconfigjson"}}' | base64 -d > ${CLUSTER_PULL_SECRET}
+jq ".auths += ${CLOUD_PULL_SECRET}" < ${CLUSTER_PULL_SECRET} > ${UPDATED_PULL_SECRET_FILE}
+oc set data secret/pull-secret -n openshift-config --from-file=.dockerconfigjson=${UPDATED_PULL_SECRET_FILE}
```

Comment:
It is only updated the secret file, but how about OCM staging env? It is using different telemetry service with prod. This will register the cluster to production only. And OCM ENV should be exposed as a variable for future testing supporting.

Reply:
ok that is good to know... if I create an envrionment var. called OCM_ENV and we set it to production, would we want the behavior of this step to be the same as it is now (use the default cloud.o.c pull secret in Prow)  And if we set OCM_ENV to staging for example, where would I get the proper pull secret for cloud.o.c (Assuming cloud.o.c is the only secret I would need to add)?  I want to make sure we aren't leaking pull secrets.